scripts/prepare_client.sh

```diff
@@ -70,7 +70,7 @@ cd "${GOPATH}/src/github.com/google/keytransparency"
 go build ./cmd/keytransparency-client
 
 # Generate .keytransparency.yaml file.
-KTYAML="log-key: \"../trillian/testdata/ct-http-server.pubkey.pem\"
+KTYAML="log-key: \"../trillian/testdata/log-rpc-server.pubkey.pem\"
```

Comment:
Isn't this supposed to be `genfiles/trillian-log.pem`?

Please add TODO to fetch this info from #672